rest-service/manager_rest/test/endpoints/test_blueprints.py

```diff
@@ -449,3 +450,19 @@ def test_force_delete_used_blueprints_via_import(self):
                           first_blueprint_id)
 
         self.client.blueprints.delete(first_blueprint_id, True)
+
+    @attr(client_min_version=base_test.LATEST_API_VERSION,
+          client_max_version=base_test.LATEST_API_VERSION)
+    def test_not_listing_hidden_blueprints(self):
+        blueprint_file = 'blueprint_with_inputs.yaml'
+        blueprint_path = os.path.join(
+            self.get_blueprint_path('mock_blueprint'),
+            blueprint_file)
+        b0_id = 'b0' + HIDDEN_BLUEPRINT_SUFFIX + 'asdf'
+        b1_id = 'b1' + HIDDEN_BLUEPRINT_SUFFIX
```

Comment:
The user should not be able to upload a hidden blueprint, because it can not be managed. Maybe change it to a flag prop